lxd/init.go

```diff
@@ -130,6 +130,77 @@ func initDataNodeApply(d lxd.InstanceServer, config api.InitLocalPreseed) (func(
 		}
 	}
 
+	// Apply storage volumes configuration.
+	applyStorageVolume := func(storageVolume api.InitStorageVolumesProjectPost) error {
+		// Get the current storageVolume.
+		currentStorageVolume, etag, err := d.UseProject(storageVolume.Project).GetStoragePoolVolume(storageVolume.Pool, storageVolume.Type, storageVolume.Name)
+
+		if err != nil {
+			// Create the storage volume if it doesn't exist.
+			err := d.UseProject(storageVolume.Project).CreateStoragePoolVolume(storageVolume.Pool, storageVolume.StorageVolumesPost)
```

Comment:
@tomponline Seems the storage volume creation is failing here.
Am I trying to create storage volumes in the default project too soon?
The storage pool should exist because we are creating it an earlier stage 114-129?

Reply:
Nevermind. Was missing type which is required.